.setup/CONFIGURE_SUBMITTY.py

```diff
@@ -239,6 +247,35 @@ def get_input(question, default=""):
 
     CGI_URL = SUBMISSION_URL + '/cgi-bin'
 
+    print("NOTE: Emails can be set up at a later time in the config/email.json file")
+    while True:
+        is_email_enabled = get_input("Will Submitty use email notifications? [y/n]", 'y').lower()
+        if (is_email_enabled == 'y') :
```

Comment:
Let's handle yes/no here (`is_email_enabled.lower() in ['yes', 'y']`, `is_email_enabled.lower() in ['no', 'n']`) as that's somewhat standard in the unix world.